src/App.tsx

```diff
@@ -1,95 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [total, setTotal] = useState(9);
+
```

Comment:
Total should be a number of total content items, not pagination buttons `items.length`
<img width="493" alt="image" src="https://user-images.githubusercontent.com/99331793/180739132-67057e6f-6d7c-45c0-9c1b-ba4acf6fa50a.png">